boltstub/simple_jolt/transformers.py

```diff
@@ -228,7 +228,7 @@ def _decode_full(value, decode_cb):
                 or (isinstance(value, list)
                     and all(isinstance(b, int) and 0 <= b <= 255
                             for b in value))):
-            raise JOLTValueError('Expected str or list of integers (0-255) '
+            raise JOLTValueError("Expected str or list of integers (0-255) "
```

Comment:
does mixing single quote and double quote strings work ok? See following line where it uses single quotes.

Reply:
Yes is does, no I don't particularly like it. I prefer: double quotes, unless you have `"` in the string + I prefer to have a consistent quotation symbol for wrapped strings like this one. The analyzer currently does not support the second aspect (https://github.com/zheller/flake8-quotes/issues/82). I could work around this by adding `# noqa: noqa: Q000` at the end of each conflicting line, but that makes the code base uglier than mixing sting styles, imo.